lib/brakeman/report/report_sonar.rb

```diff
@@ -0,0 +1,37 @@
+class Brakeman::Report::Sonar < Brakeman::Report::Base
+  def generate_report
+    return {
+      issues: all_warnings.map { |warning| issue_json(warning) }
+    }.to_json
```

Comment:
The JSON report uses `JSON.pretty_generate` instead of `to_json`, which makes the output nicely formatted.